pkg/util/iptables/iptables.go

```diff
@@ -664,6 +674,32 @@ func getIPTablesWaitFlag(version *utilversion.Version) []string {
 	}
 }
 
+// getKernelVersion returns currently running kernel version.
+func getKernelVersion() (string, error) {
+	kernelVersionFile := "/proc/sys/kernel/osrelease"
+	fileContent, err := ioutil.ReadFile(kernelVersionFile)
```

Comment:
So I guess this is the same way the IPVS code reads the kernel version, so I guess it's right? Maybe we should move this into `pkg/util/` somewhere? (Well, "somewhere other than `pkg/util/iptables`" I mean.)